src/renderers/dom/client/wrappers/__tests__/ReactDOMTextarea-test.js

```diff
@@ -24,46 +25,51 @@ describe('ReactDOMTextarea', function() {
   beforeEach(function() {
     React = require('React');
     ReactDOM = require('ReactDOM');
+    ReactDOMServer = require('ReactDOMServer');
     ReactLink = require('ReactLink');
     ReactTestUtils = require('ReactTestUtils');
 
-    renderTextarea = function(component, container) {
+    renderTextarea = function(component, container, skipReplace) {
       if (!container) {
         container = document.createElement('div');
       }
-      var stub = ReactDOM.render(component, container);
-      var node = ReactDOM.findDOMNode(stub);
-      // Fixing jsdom's quirky behavior -- in reality, the parser should strip
-      // off the leading newline but we need to do it by hand here.
-      node.value = node.innerHTML.replace(/^\n/, '');
-      return stub;
+      var node = ReactDOM.render(component, container);
+
+      if (!skipReplace) {
```

Comment:
I believe this was causing a quirk with one of the test cases, as setting `innerHTML` changed the behavior of my test. Also, I think it's a strange hack that should be opt-in instead of opt-out for test cases which call into `renderTextarea`. That said, if it's no longer applicable, you can surely remove the flag.